<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>785</width>
    <height>769</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Archerrank2</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>2</horstretch>
     <verstretch>2</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>507</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="5">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetNoConstraint</enum>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0">
        <property name="spacing">
         <number>6</number>
        </property>
        <property name="sizeConstraint">
         <enum>QLayout::SetNoConstraint</enum>
        </property>
        <item>
         <widget class="QLabel" name="label_user">
          <property name="text">
           <string>participants</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView_user">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>1</horstretch>
            <verstretch>1</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_user">
          <property name="text">
           <string>add new user</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_edituser">
          <property name="text">
           <string>edit user</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_deleteuser">
          <property name="text">
           <string>delete user</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>785</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuInfo">
    <property name="title">
     <string>I&amp;nfo</string>
    </property>
    <addaction name="actionInfo"/>
   </widget>
   <widget class="QMenu" name="menuEvaluation">
    <property name="title">
     <string>E&amp;valuation</string>
    </property>
    <addaction name="actionOverview"/>
    <addaction name="actionPrintPreview"/>
    <addaction name="actionXLSX_Export"/>
    <addaction name="separator"/>
    <addaction name="actionCreateAddress"/>
    <addaction name="actionCreateCertificates"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEvaluation"/>
   <addaction name="menuInfo"/>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>&amp;Exit</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionNew">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>&amp;Info</string>
   </property>
  </action>
  <action name="actionOverview">
   <property name="text">
    <string>&amp;Overview</string>
   </property>
  </action>
  <action name="actionPrintPreview">
   <property name="text">
    <string>&amp;Print Preview</string>
   </property>
  </action>
  <action name="actionLoad_file">
   <property name="text">
    <string>&amp;Load file</string>
   </property>
  </action>
  <action name="actionSave_file_as">
   <property name="text">
    <string>&amp;Save file as</string>
   </property>
  </action>
  <action name="actionCreateCertificates">
   <property name="text">
    <string>&amp;Create certificates</string>
   </property>
  </action>
  <action name="actionXLSX_Export">
   <property name="text">
    <string>XLSX Export</string>
   </property>
  </action>
  <action name="actionCreateAddress">
   <property name="text">
    <string>Create Address</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
